src/idl_parser.cpp

```diff
@@ -20,6 +20,12 @@
 #include <string>
 #include <utility>
 
+#ifdef FLATBUFFERS_GENERAL_HEAP_ALLOC_OK
```

Comment:
the parser does quite a lot of allocs, I don't think it will be simple to get it alloc free or even use a given allocator, so not sure what use this is.

Reply:
Same comment as above, this is just forcing the flag to be True to allow heap allocations.